libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -98,6 +98,13 @@ const AP_Param::GroupInfo AP_MotorsMulticopter::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("THR_MIX_MAX", 14, AP_MotorsMulticopter, _thr_mix_max, AP_MOTORS_THR_MIX_MAX_DEFAULT),
 
+    // @Param: PWM_TYPE
+    // @DisplayName: Output PWM type
+    // @Description: This selects the output PWM type, allowing for normal PWM continuous output or OneShot125
+    // @Values: 0:Normal,1:OneShot,2:OneShot125
+    // @User: Advanced
+    AP_GROUPINFO("PWM_TYPE", 15, AP_MotorsMulticopter, _pwm_type, PWM_TYPE_NORMAL),
```

Comment:
@tridge and I had a quick discussion about this.  It's possible that we might be able to remove the "PWM_TYPE" parameter after we add the PWM_MIN and PWM_MAX params.  PWM_TYPE of OneShot also enables outputting the pwm value immediately (instead of waiting up to 2ms) so we would just need to be sure that we're happy with this immediate pwm output feature affecting all pwm output.
I'm still planning on adding the PWM_MIN, MAX in the near future.